distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -905,7 +921,7 @@ type Table
                 _ -> type_mapping.value_type_to_sql argument_value_type Problem_Behavior.Ignore
             expr = SQL_Expression.Constant value
             new_type_ref = SQL_Type_Reference.from_constant sql_type
-            Column.Value ("Constant_" + UUID.randomUUID.to_text) self.connection new_type_ref expr self.context
+            Column.Value ("Constant_" + value.to_text) self.connection new_type_ref expr self.context
```

Comment:
When writing the ticket, I thought this would be the place where we'd actually be using the unique name strategy. To ensure that two distinct columns that represent the same constant would get a distinct name.

After thinking a bit how the SQL is generated, I guess we are safe without additional checks - 2 constant columns that are 'internal' to the expression (like in the example I suggested to add to the tests) are OK - because when building the expression we only extract the `SQL_Expression` part and do not really use the names of the internal 'sub-columns' anywhere - only the name of the overall resulting column added to the table is meaningful.

So looks like this is fine.